File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.py
class Solution:
    def reverseDegree(self, s: str) -> int:
        str="Azyxwvutsrqponmlkjihgfedcba"
        res=[]
        count=0
        for i in s:
            count+=1
            if i in str:
                a=str.index(i)*count
                res.append(a)
        return sum(res)        
        

File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.java
class Solution {
    public int reverseDegree(String s) {
        int count=0;
        for(int i=0;i<s.length();i++){
           int value ='z'-s.charAt(i)+1;
            count=count+value*(i+1);
           }
        
    return count;
}
}